src/libcore/default.rs

```diff
@@ -115,6 +134,35 @@ pub trait Default: Sized {
     fn default() -> Self;
 }
 
+/// A helper function that, when imported, allows one to reduce duplication, by
+/// saying just `default()` instead of `Default::default()`, when the compile
+/// can infer the value type.
```

Comment:
It feels like a lot of parts of this sentence carry very little information. If there is information you feel is necessary, please add it, otherwise please rephrase this more concisely. Maybe this:

> Equivalent to `Default::default()`.

Reply:
Used your second suggestion.